baybe/kernels/basic.py

```diff
@@ -0,0 +1,46 @@
+"""Collection of kernels."""
+
+from fractions import Fraction
+from typing import Union
+
+from attrs import define, field
+from attrs.validators import in_
+
+from baybe.kernels.base import Kernel
+
+
+def _convert_fraction(value: Union[str, float, Fraction], /) -> float:
+    """Convert the provided value into a float.
+
+    Args:
+        value: The parameter that should be converted.
+
+    Returns:
+        The converted parameter.
+
+    Raises:
+        ValueError: If `value` was provided as string but could not be interpreted as
+            fraction.
```

Comment:
```suggestion
        ValueError: If the input was provided as string but could not be interpreted as
            fraction.
```
Good that you added the `/` to the signature, but the whole point of this is that the name of the parameter is hidden from the user. So it makes no sense to reference it in the doc/error message